src/instructlab/model/train.py

```diff
@@ -87,29 +165,67 @@
         "(reduces GPU VRAM usage and may slow down training)"
     ),
 )
+@click.option("--max-seq-len", type=int)
```

Comment:
This setting determines the maximum length (in tokens) of a SINGLE sample. Think of this as a filter that drops all samples (doesn't train on them) from the dataset exceeding this length.